astropy/units/quantity_helper/erfa.py

```diff
@@ -3,15 +3,39 @@
 """Quantity helpers for the ERFA ufuncs."""
```

Comment:
Are any of these publicly scoped? Nothing has docstrings...

Reply:
No, they're not. There are some comments about what these functions are supposed to do in `quantity_helpers/helpers.py`.  I think a good docstring in `quantity_helpers/__init__.py` would be an idea...  (I've long wondered whether we should use module docstrings for that type of information on implementation; #8930)

Reply:
Seems reasonable. It's where I write that, when I remember to.
I looked at #8930. I like everything except the custom section title. Numpydoc will complain unless we register in a new section title. The existing section titles seems sufficient, `Notes`` or ``routine listings`` can go into details. Or just stuff in the extended summary.